masonry/src/widgets/text_area.rs

```diff
@@ -848,23 +872,28 @@ impl<const EDITABLE: bool> Widget for TextArea<EDITABLE> {
             self.editor.try_layout().unwrap()
         };
         if ctx.is_focus_target() {
+            let (window_is_focused, selection_color) = if ctx.is_window_focused() {
+                (true, props.get::<SelectionColor>().color)
+            } else {
+                (false, props.get::<UnfocusedSelectionColor>().0.color)
+            };
             for (rect, _) in self.editor.selection_geometry().iter() {
-                // TODO: If window not focused, use a different color
-                // TODO: Make configurable
                 scene.fill(
                     Fill::NonZero,
                     Affine::IDENTITY,
-                    palette::css::STEEL_BLUE,
+                    selection_color,
                     None,
                     &rect,
                 );
             }
-            if let Some(cursor) = self.editor.cursor_geometry(1.5) {
-                // TODO: Make configurable
+            if let Some(cursor) = self.editor.cursor_geometry(1.5)
+                && window_is_focused
+                && self.t <= 0.6
+            {
                 scene.fill(
                     Fill::NonZero,
                     Affine::IDENTITY,
-                    palette::css::WHITE,
+                    props.get::<CaretColor>().color,
```

Comment:
Our style is for all `props.get` calls to be on their own line (i.e. `let caret_color = props.get::<CatetColor>()`) . I believe this is to make it easier to validate that `properties_changed` implementation.
It isn't clear to me that this is entirely necessary, but it would be better to maintain that for now.